ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -727,7 +729,7 @@ private final class OracleQueries(tablePrefix: String) extends Queries(tablePref
   protected[this] override def agreementTextType = sql"NCLOB"
 
   protected[this] override def jsonColumn(name: Fragment) =
-    sql"$name CLOB NOT NULL CONSTRAINT ensure_json_$name CHECK ($name IS JSON)"
+    sql"$name CLOB NOT NULL CONSTRAINT ${tablePrefixFr}ensure_json_$name CHECK ($name IS JSON)"
```

Comment:
So this was one bug I assume? You only hit this if you actually set a prefix right?

Reply:
This is the bug which only appears if you run two instances of the json api with different prefixes.